CMakeLists.txt

```diff
@@ -126,10 +129,38 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 #
 # cmake ... -DCMAKE_INSTALL_PREFIX=$HOME/opt
 #
-include(GNUInstallDirs)
 if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
   set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE PATH "..." FORCE)
 endif()
+include(GNUInstallDirs)
+
+
+# Configure how the SDK will manage runtime paths. This is important
+# even for BUILD_SHARED_LIBS=OFF due to the always-shared
+# `libviam_rust_utils` shared library.
+set(CMAKE_BUILD_WITH_INSTALL_RPATH false)
```

Comment:
These five flags basically set things up so that we have valid rpaths in both the build tree and the install tree, and that the installation tree is relocatable (e.g. uses $ORIGIN or @loader_path / @rpath).